web/template/watch.gohtml

```diff
@@ -13,297 +13,345 @@
+                            <div class="text-right"
+                                 x-text="(Math.round(data.bufferSeconds * 100) / 100) + 's'"></div>
+                            <div class="text-right" x-text="data.mediaRequests"></div>
+                            <div class="text-right" x-text="data.mediaRequestsFailed"></div>
+                        </div>
                     </div>
-                </div>
 
+                </div>
             </div>
         </div>
-        <div id="streamInfo" class="mx-5 m-auto divide-y dark:divide-gray-800 divide-solid space-y-4 space-y-reverse">
-            <!-- Title, course name, actions -->
-            <div class="m-auto pb-2">
-                <div class="flex justify-between align-middle mt-4 m-auto">
-                    <h1 class="text-4 text-xl grow m-auto" @titleupdate.window="$el.innerText=$event.detail.title">
+        <!-- Chat -->
+        {{if and $course.ChatEnabled $stream.ChatEnabled}}
+            <div x-show="showChat"
+                 :class="showChat ? 'lg:basis-1/4' : 'lg:basis-0'"
+                 class="z-40 basis-full order-3 border rounded p-1 mx-2 bg-gray-100 lg:border-0 lg:shadow lg:order-none lg:mx-0 lg:p-0 lg:bg-transparent">
+                <div id="chat-box" class="bg-white rounded lg:bg-transparent md:rounded-0">
+                    {{template "chat" .ChatData}}
+                </div>
+            </div>
+        {{end}}
+
+        <!-- stream info container -->
+        <!-- Title, course name, actions -->
+        <div class="basis-full relative p-5 lg:order-none order-2">
+            <div class="flex justify-between align-middle">
+                <!-- title and course-name -->
+                <div class="grid mb-auto">
+                    <h1 class="font-bold text-4 text-2xl grow"
+                        @titleupdate.window="$el.innerText=$event.detail.title">
                         {{if $stream.Name}}
                             {{$stream.Name}}
                         {{else}}
                             Lecture: {{$stream.Start.Month}} {{printf "%02d." $stream.Start.Day}} {{$stream.Start.Year}}
                         {{end}}
                     </h1>
-                    <!-- Desktop stream info -->
-                    <span class="space-x-2 text-lg text-4 hidden sm:inline-flex shrink-0">
+                    <div class="flex justify-between flex-col sm:flex-row">
+                        <div class="flex my-auto">
+                            <a href="/course/{{$course.Year}}/{{$course.TeachingTerm}}/{{$course.Slug}}">
+                                <span class="text-5 hover:text-1 hover:underline">{{$course.Name}}</span></a>
+                            {{if or (.IndexData.TUMLiveContext.User.IsAdminOfCourse .IndexData.TUMLiveContext.Course) .IndexData.IsAdmin}}
+                                <a class="hover:bg-gray-200 dark:hover:bg-gray-600 text-5 hover:text-1 w-fit m-2 px-2 inline-block rounded my-auto"
```

Comment:
I am not really a fan of the look of this floating icon. Maybe we can just show it next to the other stream action icons? I think it would make sense as it just another way to interact with the watch page.

<img width="252" alt="grafik" src="https://user-images.githubusercontent.com/30267166/192624305-fa087819-52a9-42b2-bee5-e839c35d1560.png">

Maybe just left of the download icon?

Reply:
Moved it to the left. Great idea, btw!

<img width="287" alt="image" src="https://user-images.githubusercontent.com/29633518/193505289-6f1ca180-2059-4b9d-9b4e-7192b08c3098.png">

Reply:
Looks perfect, much more tidy imo.